test/e2e-common.sh

```diff
@@ -258,6 +258,27 @@ function install_knative_serving_standard() {
   # If we need to build from source, then kick that off first.
   if [[ -z "$1" ]]; then
     build_knative_from_source
+  else
+    SERVING_ISTIO_YAML="${1}"
+  fi
+
+  echo ">> Installing Knative namespace and CRD"
+  echo "Namespace YAML: ${SERVING_NAMESPACE_YAML}"
+  echo "CRD YAML: ${SERVING_CRD_YAML}"
+  kubectl apply \
+	  -f "${SERVING_NAMESPACE_YAML}" \
+	  -f "${SERVING_CRD_YAML}" || return 1
```

Comment:
I don't think that this works for the case where folks have brought their own file (I think that the upgrade test might do that?) 🤔

Reply:
There is a way of using the overall bundle and filtering the CRDs, which is probably what you want to do, but you may need an if/else like below (maybe fold it into the if/else above?)

Reply:
Yes, it sounds great. But namespace does not have the label to filter. So, I added a new label, but is there any good idea for namespace?